xpti/include/xpti/xpti_trace_framework.hpp

```diff
@@ -1731,6 +1731,7 @@ class tracepoint_t {
     m_default_event_type = (uint16_t)xpti::trace_event_type_t::algorithm;
     m_default_activity_type = xpti::trace_activity_type_t::active;
     m_default_name = "Message"; // Likely never used
+    m_instID = 0;
```

Comment:
Coverity was complaining about this field be left uninitialized in the constructor.